pyproject.toml

```diff
@@ -31,6 +31,8 @@ rabbit = ["pika"]
 kafka = ["kafka-python"]
 azure = ["azure-storage-blob", "cffi"]
 
+all = ["redis", "pika", "kafka-python", "azure-storage-blob", "cffi"]
```

Comment:
Is there no way to nest extras?

Reply:
https://github.com/python-poetry/poetry/issues